src/AWSClusterManagersDemo.jl

```diff
@@ -0,0 +1,44 @@
+using AWSClusterManagers: AWSBatchManager
+using Distributed
+
+const NUM_WORKERS = 4
+
+# Must start workers before any @everywhere for them to be available on the workers
+addprocs(AWSBatchManager(NUM_WORKERS))
+
+@everywhere function throw_darts()
+    MAX_THROW = 250_000
+    dart_count = Integer(floor(rand() * MAX_THROW))
+    hits = 0
+
+    for _ in 1:dart_count
+        x = rand()
+        y = rand()
+
+        if ((x*x) + (y*y) < 1)
+            hits += 1
+        end
+    end
+
+    return hits, dart_count
+end
+
+calc_approx_pi(hits::Integer, thrown::Integer) = 4 * hits / thrown
+
+function main()
+    total_hits = 0
+    total_throws = 0
+
+    for worker in workers()
+        hits, throws = remotecall_fetch(throw_darts, worker)
```

Comment:
using `remotecall_fetch` will mean you run nothing in parallel at all.
To run them in parallel use 
```
futures = [remotecall(throw_dates, worker) for worker in workers]
for (worker, future) in enumerate(futures)
    hits, throws = fetch(future)
    ....
end 
```

Reply:
Yeah, this was more showing the distributed part of ClusterManagers to just spawn a worker, and execute a task.
I'll modify to this though!